packages/components/psammead-image/src/index.jsx

```diff
@@ -32,13 +32,45 @@ const StyledImg = styled.img`
   width: 100%;
 `;
 
+const getMimeType = srcset => {
+  if (!srcset) return null;
+
+  const firstSrcsetUrl = srcset.split(',')[0].split(' ')[0];
```

Comment:
just wondering if this is easier to work out in simorgh

Reply:
I think its much of a muchness really. It would probably be a bit more work in Simorgh to pass through a `mimeType` prop or something similar to the Psammead component, with similar checking done in Simorgh. Conscious not having too much rendering logic in Psammead though.